src/test/java/model/LottoCreationTest.java

```diff
@@ -0,0 +1,26 @@
+package model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+class LottoCreationTest {
+
+    private final LottoCreation lottoCreation = new LottoCreation();
+
+    @Test
+    @DisplayName("전달받은 개수만큼 복권을 생성할 수 있다.")
+    void canCreateLotto() {
+        // given
+        final int number = 10;
+
+        // when
+        final List<Lotto> lotto = lottoCreation.createLotto(number);
+
+        // then
+        assertThat(lotto.size()).isEqualTo(number);
```

Comment:
갯수를 확인하는 메서드는 hasSize 를 쓰면됩니다.

```suggestion
        assertThat(lotto).hasSize(number);
```

Reply:
안 그래도 sonarLint에서 hasSize로 변경을 권해서 시도했었는데 계속 오류가 떠서 그냥 제출했습니다.
여전히 오류가 나는 상황입니다 ㅠㅠ
![image](https://user-images.githubusercontent.com/81404890/219001007-039f98a0-b792-47e3-a123-5e5b15338528.png)